pi_processing/processors.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+function error_section_value_error_between(
+    string $parameter,
+    string $min,
+    string $max
+): string {
+    $min_tag_name = is_numeric($min) ? 'literal' : 'constant';
+    $max_tag_name = is_numeric($max) ? 'literal' : 'constant';
+
+    $min_tag = '<' . $min_tag_name . '>' . $min . '</' . $min_tag_name . '>';
+    $max_tag = '<' . $max_tag_name . '>' . $max . '</' . $max_tag_name . '>';
+
+    return str_replace(
```

Comment:
Not sure if I know what I am talking about, but wouldn't `strtr` make more sense here?

Reply:
No, str_replace() with arrays is what I want.